src/backend/InvenTree/company/models.py

```diff
@@ -56,9 +54,21 @@ def rename_company_image(instance, filename):
     return os.path.join(base, fn)
 
 
+class CompanyReportContext(report.mixins.BaseReportContext):
+    """Report context for the Company model.
+
+    Attributes:
+        name: The name of the Company
+        primary_address: The primary address associated with the Company
+    """
+
+    name: str
```

Comment:
Some attributes are missing here, and why not expose the complete company object?

Reply:
See https://github.com/inventree/InvenTree/pull/9819